acceptance/ui/features/steps/generic_steps.rb

```diff
@@ -122,4 +135,38 @@ def sortColumn(category, sortOrder)
     while categoryLink[:class] != order do
         categoryLink.click()
     end
+end
+
+def removeAllEntries()
+    entries = page.all("[ng-repeat$='in $data']")
+    for i in 0..(entries.size - 1)
+        within(entries[i]) do
+            click_link_or_button("Delete")
+        end
+        click_link_or_button("Remove")
+    end
+end
+
+def getTableValue(valueOrTableUrl)
+    if valueOrTableUrl.start_with?("table://") == false
+        return valueOrTableUrl
+    end
+    parsedUrl = valueOrTableUrl.split(/\W+/)
+    tableType = parsedUrl[1]
+    fileName = File.join(ENV["DATASET_FILES"], tableType + ".json")
```

Comment:
Instead of reading/parsing the files on every call to getTableValue(), can you refactor this a bit so that at startup (somewhere in env.rb), you read/parse DATASET_FILES/*.json into a some kind of singleton for `table`? 

This will save a little on I/O since we call getTableValue() so many times.
If you can not read or parse one of the files, then the test should exit with a fatal error.